docs/_templates/template_ft_in_action.md

```diff
@@ -6,27 +6,28 @@ This is a template for the documentation guides of Finetuner in action, with the
 For more info on best practices for documentation, see Jina's [contribution guidelines](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-contributing-documentation)
 ```
 
-## Task overview
+## Task
 Describe the task which this guide accomplishes, including which model will be fine-tuned and which dataset you will use.
 
 Also provide a brief description of what the task entails, what the dataset looks like and a high-level description of how the dataset is processed.
 
 
-## Preparing data
-Outline where the data can be found, artifact names in hubble or if relevant, how a user might load their own custom data. 
+## Data
+Outline where the data can be found for your particular task, artifact names in hubble or if relevant.
+
 Add a link to supplementary dataset info, for example as a `See Also` {admonition}.
 If you are outlining how to preprocess a dataset from scratch, use {dropdown} to hide long code snippets.
 
 
-## Choosing the model
+## Backbone model
 Mention which model will be used in your fine-tuning task. Feel free to add a `See Also` {admonition} for supplementary info on the model, perhaps a relevant paper or site.
 
 You can also add a `Tip` {admonition} for how the user can view all available models, also referring to the `Choose backbone model` documentation.
 
 
-## Creating a fine-tuning run
+## Fine-tuning
```

Comment:
One important thing is that the code, when combined, should be runnable. So we should add a subsection with finetuner.login() inside `Fine-tuning` showing how to login to jina ecosystem.

Reply:
Cant this `finetuner.login()` just be added to your code example in the fine-tuning section and then add a warning that users must login? There is a whole section `Login to Finetuner` in the `Step by step` so I don't think the guides need to have a separate section for this.